src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.OpenSsl.cs

```diff
@@ -338,9 +338,10 @@ private static void QueryUniqueChannelBinding(SafeSslHandle context, SafeChannel
         private static IntPtr GetSslMethod(SslProtocols protocols)
         {
             Debug.Assert(protocols != SslProtocols.None, "All protocols are disabled");
-
+#pragma warning disable 0618
```

Comment:
is 618 for obsolete? if so perhaps just add a small comment to note why are we adding this supression.